jaxrs-api/src/main/java/jakarta/ws/rs/core/Form.java

```diff
@@ -93,4 +93,29 @@ public Form param(final String name, final String value) {
     public MultivaluedMap<String, String> asMap() {
         return parameters;
     }
+
```

Comment:
As this file has a two-year copyright note, the EF's rules require that you update it. To make future updates easier, it is best if you simply change to to a one-year copyright note.